CONTRIBUTING.md

```diff
@@ -29,17 +28,9 @@ On macOS, we recommend using `brew` to install Python. For Windows, we recommend
 
 ### Fork the Repo
 
-Go to the [home page of the repo](https://github.com/getsentry/sentry-python), click the "Fork" button, and go through the process of creating your fork of `sentry-python`.
+Before you can contribute, you will need to [fork the `sentry-python` repository](https://github.com/getsentry/sentry-python/fork).
 
-### Clone the Forked Repo
-
-You should now have a new `sentry-python` repo in your GitHub projects. Clone it to work on it locally:
-
-```bash
-git clone git@github.com:<YOUR-USERNAME>/sentry-python.git
-```
-
-Replace `<YOUR-USERNAME>` with your actual GitHub username. Alternatively, you can go to your GitHub profile, find the forked `sentry-python` repo, and copy the clone URL from there.
+Then, clone the forked repository to your local development environment.
```

Comment:
I would suggest merging these two sentences into a single line. And, maybe we can rename the section to "Fork and Clone the Repository" or something similar